Update zero-downtime doc to add health check aggressiveness note and fix broken links (#6438)
solo-io/gloo: docs/content/operations/advanced/zero-downtime-gateway-rollout/_index.md

  ignoreHealthOnHostRemoval: true
{{< /highlight >}}

> Some customers have noticed that the above active health check settings are too aggressive for their environment, with `Upstream` pings being issued every 2 seconds. If that is true for you and the health checks themselves generate excessive traffic, then consider less aggressive active health check settings, like these:

Reviewer: Unless you think people might search for "aggressive," I might avoid using that term and instead use the same term as the setting (interval). 
```suggestion
In the previous example, `Upstream` pings are issued every 2 seconds. You might find that this active health check setting is too frequent and generates excessive traffic. If so, consider a health check with a longer interval, such as the following example.
```

{{< highlight yaml "hl_lines=6-9" >}}
  # ----- Health Check (a.k.a. active health checks) -------
  healthChecks:
    - healthyThreshold: 1
      httpHealthCheck:
        path: /status/200
      interval: 15s
      noTrafficInterval: 10s
      timeout: 5s
      unhealthyThreshold: 3
      reuseConnection: false
```
{{< /highlight >}}

> Full documentation of the health check API options is available [here]({{% versioned_link_path fromRoot="/reference/api/github.com/solo-io/gloo/projects/gloo/api/external/envoy/api/v2/core/health_check.proto.sk/#healthcheck" %}}).

Reviewer: ```suggestion
For more information, see the [health check API documentation]({{% versioned_link_path fromRoot="/reference/api/github.com/solo-io/gloo/projects/gloo/api/external/envoy/api/v2/core/health_check.proto.sk/#healthcheck" %}}).
```
